src/sample-account-setup.test.ts

```diff
@@ -1,27 +1,546 @@
+export async function handler(medplum: MedplumClient, event: BotEvent): Promise<any> {
+  const patient = event.input as Patient;
 
-test('Create data for patient', async () => {
-  const medplum = new MockClient();
+  const date = new Date();
 
-  const input = await medplum.createResource<Patient>({
-    resourceType: 'Patient',
-    name: [{ given: ['Michael'], family: 'Scott' }],
-    telecom: [
+  const getPreviousDay = (date = new Date()): Date => {
+    const previous = new Date(date.getTime());
+    previous.setDate(date.getDate() - 1);
+
+    return previous;
+  };
+
+  const subject: Reference<Patient> = {
+    reference: createReference(patient).reference,
+    display: getDisplayString(patient),
+  };
+
+  //Create a practitioner if if isn't already created
+  const practitioner = await medplum.createResourceIfNoneExist<Practitioner>(
+      {
+        resourceType: 'Practitioner',
+        id: 'practitioner',
+        name: [
+          {
+            given: ['Alice'],
+            family: 'Smith',
+          },
+        ],
+        photo: [
+          {
+            contentType: 'image/png',
+            url: 'https://docs.medplum.com/img/cdc-femaledoc.png',
+          },
+        ],
+      },
+      'Practitioner/practitioner'
+  );
+
+  patient.generalPractitioner = [createReference(practitioner)];
+  await medplum.updateResource(patient);
+
+  //Create a schedule for the practitioner
+  const schedule = await medplum.createResourceIfNoneExist<Schedule>(
+      {
+        resourceType: 'Schedule',
+        id: 'schedule',
+        actor: [
+          {
+            reference: createReference(practitioner).reference,
+            display: getDisplayString(practitioner),
+          },
+        ],
+      },
+      'Schedule/schedule'
+  );
+
+  //Populate the schedule with slots
+  const slots: Slot[] = [];
+  const slotDate = new Date();
+  for (let day = 0; day < 60; day++) {
+    for (const hour of [9, 10, 11, 13, 14, 15]) {
+      slotDate.setHours(hour, 0, 0, 0);
+      const slot = await medplum.createResourceIfNoneExist<Slot>(
+          {
+            resourceType: 'Slot',
+            id: `slot-${day}-${hour}`,
+            start: slotDate.toISOString(),
+            schedule: createReference(schedule),
+          },
+          'Slot/slot-' + day + '-' + hour
+      );
+      slots.push(slot);
+    }
+    slotDate.setDate(date.getDate() + 1);
+  }
+
+  //Create observations for DiagnosticReport if not exits
+  const hemoglobinA1c = await medplum.createResourceIfNoneExist<Observation>(
+      {
+        resourceType: 'Observation',
+        id: 'hemoglobinA1c',
+        subject,
+        code: {
+          text: 'Hemoglobin A1c',
+        },
+        valueQuantity: {
+          value: 5.4,
+          unit: 'mmol/L',
+        },
+        referenceRange: [
+          {
+            high: {
+              value: 7.0,
+            },
+          },
+        ],
+      },
+      `Observation/hemoglobinA1c`
```

Comment:
This also needs to be related to patient.  I'm not sure the best way to handle this, but will ask for guidance from @codyebberson and @rahul1 - namely this should be something like `Observation\patient.id+hemoglobinA1c